docs/vhost-user-blk-testing.md

```diff
@@ -0,0 +1,102 @@
+# How to test Vhost-user block with SPDK
+
+The purpose of this document is to illustrate how to test vhost-user-blk in cloud-hypervisor with SPDK as the backend.
+
+## Framework
+
+It's a simple test to validate the block read/write between VM and block backend.
+```
+             +----+----------+          +-------------+-----------+
+             |    |          |          |             |           |
+             |    |vhost-user|----------| vhost-user  |    dpdk   |
+             |    |blk device|          | port 1      |           |
+             |    |          |          |             |           |
+             |    +----------+          +-------------+-----------+
+             |               |          |                         |
+             |      vm       |          |           spdk          |
+             |               |          |                         |
+          +--+----------------------------------------------------+--+
+          |  |                  hugepages                         |  |
+          |  +----------------------------------------------------+  |
+          |                                                          |
+          |                       host                               |
+          |                                                          |
+          +----------------------------------------------------------+
+```
+## Prerequisites
+
+Prior to running the test, the following steps need to be performed.
+- Enable hugepages
+- Install SPDK
+
+Here are some good references for detailing them.
+- spdk
+	* https://spdk.io/doc/
+
+## Test environment
+
+The below test environment is based on ubuntu release, as for other system, please check related document.
```

Comment:
Which Ubuntu release.

Reply:
My Ubuntu release is Ubuntu 16.04.1 LTS, 

From @uMartinXu's test, his release is Ubuntu 18.04.3 LTS.

So,  which release is more suitable here? thanks.